app/tasks/raster_tile_cache_assets/symbology.py

```diff
@@ -73,14 +112,28 @@ async def pixetl_symbology(
             "calc": None,
             "resampling": PIXETL_DEFAULT_RESAMPLING,
             "grid": f"zoom_{zoom_level}",
-            "pixel_meaning": pixel_meaning,
         },
     )
+
+    # Needed by gdaldem down the line
+    source_asset_uri = tile_uri_to_tiles_geojson(
+        get_asset_uri(
+            dataset,
+            version,
+            AssetType.raster_tile_set,
+            creation_options.dict(by_alias=True),
+            "epsg:3857",
+        )
+    )
+
+    pixel_meaning = f"{source_asset_co.pixel_meaning}_{source_asset_co.symbology.type}"
```

Comment:
this will be the final tile set before running through the tiler. In my latest PR I give those tiles the same name as tile cache implementation. 

No need for action here. I can fix that when merging into my.